css/style.css

```diff
@@ -0,0 +1,892 @@
+  content: '';
+  display: block;
+  position: absolute;
+  background-image: url(../img/underline.svg);
+  background-repeat: no-repeat;
+  width: 60px;
+  height: 2px;
+  margin-top: 8px;
+}
+
+/* section 1 */
+
+.intro {
+  height: 760px;
+}
+
+.intro__wrapper {
+  background-image: url(../img/tesla_bg.svg);
+  background-repeat: no-repeat;
+  background-position: center top;
+  background-size: contain;
+  max-width: 100%;
+  height: 467px;
+}
+
+.intro__title {
+  font-weight: 600;
+  font-size: 90px;
+  line-height: 100px;
+  text-align: center;
+  margin: 0 auto;
+}
+
+.intro__subtitle {
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 18px;
+  text-align: center;
+  margin: 0 auto;
+  margin-bottom: 26px;
+}
+
+.intro__img {
+  margin: 0 273px 26px 199px;
+}
+
+.tesla {
+  max-width: 100%;
+}
+
+.intro-info {
+  margin: 0 321px 37px 321px;
+}
+
+.intro-info__inner, .intro-info__feature, .feature {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.feature {
+  width: 375px;
+}
+
+.info-title {
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 24px;
+  margin: 0;
+}
+
+.info-title > span {
+  font-weight: 600;
+  font-size: 35px;
+  line-height: 43px;
+  margin-right: 6px;
+}
+
+.info-subtitle {
+  font-weight: normal;
+  font-size: 12px;
+  line-height: 15px;
+  margin: 0;
+}
+
+.icon01 {
+  background-image: url(../img/icon01.svg);
+  background-repeat: no-repeat;
+  background-position: center top;
+  margin-right: 14px;
+  width: 46px;
+  height: 34px;
+}
+
+.icon02 {
+  background-image: url(../img/icon02.svg);
+  background-repeat: no-repeat;
+  background-position: center top;
+  width: 33px;
+  height: 50px;
+  margin-right: 11px;
+}
+
+.icon03 {
+  background-image: url(../img/icon03.svg);
+  background-repeat: no-repeat;
+  background-position: center top;
+  width: 22px;
+  height: 22px;
+  margin-right: 7px;
+}
+
+.order-title {
+  font-weight: bold;
+  font-size: 13px;
+  line-height: 16px;
+  text-transform: uppercase;
+  margin-right: 40px;
+  text-shadow: 0 3px 4px rgba(0, 0, 0, 0.3);
+}
+
+.btn {
+  width: 160px;
+  height: 40px;
+  border: 2px solid #E51937;
+  color: #E51937;
+  background: #fff;
+  font-family: montserrat;
+  font-weight: bold;
+  font-size: 13px;
+  line-height: 16px;
+  text-transform: uppercase;
+  transition: background-color 0.3s ease-out;
+}
+
+.btn:hover {
+  background-color: #E51937;
+  color: #fff;
+}
+
+.intro-arrow {
+  text-align: center;
+}
+
+/* section 2 */
+
+.about {
+  height: 900px;
+  position: relative;
+}
+
+.about__wrappper {
+  background-image: url(../img/img2.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 900px;
+  max-width: 100%;
+}
+
+.about-title {
+  position: absolute;
+  font-weight: 600;
+  font-size: 70px;
+  line-height: 85px;
+  color: #E51937;
+  left: 80px;
+  margin-top: 30px;
+}
+
+.box {
+  width: 440px;
+  height: 900px;
+  right: 0;
+  background: #E51937;
```

Comment:
Codacy found an issue: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/finalproject/pullRequest?prid=5923515)